ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx

```diff
@@ -131,6 +137,31 @@ const payUrlSuffix = "/v3/webview/transactions/pay";
 const webViewExitPathName = "/v3/webview/logout/bye";
 const webViewOutcomeParamName = "outcome";
 
+const PaymentMethodCard = (props: {
+  paymentMethod: PaymentMethod | undefined;
+}) => {
+  const { paymentMethod } = props;
+  if (paymentMethod === undefined) {
+    return null;
+  }
+  switch (paymentMethod.kind) {
+    case "CreditCard":
+      return <CreditCardComponent creditCard={paymentMethod} />;
+    case "PayPal":
+      if (payPalEnabled) {
+        return <PaypalCard paypal={paymentMethod} />;
+      }
+      return null;
+    // those methods can't pay
+    case "Satispay":
+    case "Bancomat":
+    case "BPay":
+    case "Privative":
+    default:
+      return null;
+  }
```

Comment:
```suggestion
  const { paymentMethod } = props;
  switch (paymentMethod?.kind) {
    case "CreditCard":
      return <CreditCardComponent creditCard={paymentMethod} />;
    case "PayPal":
      if (payPalEnabled) {
        return <PaypalCard paypal={paymentMethod} />;
      }
      return null;
    // those methods can't pay
    case "Satispay":
    case "Bancomat":
    case "BPay":
    case "Privative":
    case undefined:
      return null;
  }
```
I suggest to remove the `default` in order to make this component future proof (otherwise we will not receive an error when we'll add a new kind of payment method) in order to do an explicit handling.

In addition to this I've tried to avoid the first `undefined` check using the optional chaining and a new `case` but feel free to ignore this suggestion if you don't like.

Reply:
I believe it's an improvement. Good catch and thank you